geoarrow-pyarrow/src/geoarrow/pyarrow/_compute.py

```diff
@@ -562,3 +562,26 @@ def point_coords(obj, dimensions=None):
         return (pa.chunked_array(dim) for dim in zip(*flattened))
     else:
         return obj.storage.flatten()
+
+
+def to_geopandas(obj):
+    """Extract point coordinates into separate arrays or chunked arrays.
+
+    >>> import geoarrow.pyarrow as ga
+    >>> array = ga.as_geoarrow(["POINT (0 1)"])
+    >>> ga.to_geopandas(array)
+    0    POINT (0.00000 1.00000)
+    dtype: geometry
+    """
+    import pandas as pd
+    import geopandas
+
+    # Ideally we will avoid serialization via geobuffers + from_ragged_array()
```

Comment:
Do you have a higher level API than `geobuffers`? I.e. in pyarrow it's nice to have the high level nested structure, so that you don't have to rely on the implicit ordering of `geobuffers`. Like calling `flatten()` repeatedly to get the coord child of a polygon array https://github.com/geoarrow/geoarrow-python/pull/2/files#diff-75d94f02515a12ee264fbdae96bd8a630bf8bdc0e7fb9d64bec824594518d5d8R49-R51